kubernetes_deploy/staging/deployment.yml

```diff
@@ -33,20 +33,25 @@ spec:
         - name: DB_USERNAME
           valueFrom:
             secretKeyRef:
-              name: db
-              key: username
+              name: rds
+              key: database_username
+        - name: DB_NAME
+          valueFrom:
+            secretKeyRef:
+              name: rds
+              key: database_name
```

Comment:
What do you think about `db.schema`?

Reply:
Would you also change the env var name? Otherwise, I think I prefer going back to the original names (`username`, `port`, `password` etc, without the new prefix) and thus `name` here. Django calls this attribute `name`, so unless you think it's confusing then I think that'd be my preference for consistency.

Reply:
Oh Django uses `USER` not `USERNAME` so I'll change that while I'm there

Reply:
👍 Sounds good